frontend/components/SetupPage/Create/SetupBridgeOnboarding/SetupBridgeOnboarding.tsx

```diff
@@ -32,6 +35,23 @@ export const SetupBridgeOnboarding = () => {
     [setTransferAndReceivingAmounts],
   );
 
+  // If retry outcome is set, we need to update the bridge state
+  useEffect(() => {
+    if (!bridgeRetryOutcome) return;
+
+    switch (bridgeRetryOutcome) {
+      case 'NAVIGATE_TO_REFILL': {
```

Comment:
I'd rather call it `NEED_REFILL` not to connect with the UI. imagine we decide to have it all on one page and there's no navigation, this naming will be confusing